File: MPP_MusicFestival/domain/Artist.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MPP_MusicFestival.Domain
{
    class Artist : Entity<long>
    {
        public string firstName { get; set; }
        public string lastName { get; set; }
        public int age { get; set; }
        public string originCountry { get; set; }

        public Artist(long id, string firstName, string lastName, int age, string originCountry) : base(id)
        {
            this.firstName = firstName;
            this.lastName = lastName;
            this.age = age;
            this.originCountry = originCountry;
        }

        public override string ToString()
        {
            return "Artist- Nume:" + lastName + " Prenume:" + firstName +
                " Varsta:" + age + " Tara origine:" + originCountry;
        }
    }




}

File: MPP_MusicFestival_Java/FestivalModel/src/main/java/domain/Artist.java
package domain;

import java.io.Serializable;


public class Artist extends Entity<Long> implements Serializable {
    private Long id;
    private String firstName;
    private String lastName;
    private int age;
    private String originCountry;


    public Artist() {

    }

    public Artist(Long id, String firstName, String lastName, int age, String originCountry) {
        setId(id);
        this.firstName = firstName;
        this.lastName = lastName;
        this.age = age;
        this.originCountry = originCountry;
    }

    public Artist(String firstName, String lastName, int age, String originCountry) {
        setId(0L);
        this.firstName = firstName;
        this.lastName = lastName;
        this.age = age;
        this.originCountry = originCountry;
    }


    public Long getId() {
        return super.getId();
    }

    public void setId(Long id) {
        super.setId(id);
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public String getOriginCountry() {
        return originCountry;
    }

    public void setOriginCountry(String originCountry) {
        this.originCountry = originCountry;
    }


    @Override
    public String toString() {
        return "Artist{" +
                "firstName='" + firstName + '\'' +
                ", lastName='" + lastName + '\'' +
                '}';
    }
}

